philu_actions/scripts/dependency_downloader.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
```

Comment:
If we haven't already, may we check with @NIXKnight? there might be some more sophisticated way to install edx system dependencies, written in ancible playbooks or configurations?

Reply:
I have taken this script from one of the [ansible script by Edx](https://raw.githubusercontent.com/edx/configuration/master/util/install/ansible-bootstrap.sh), and customized it as per our (& Github actions) requirements